libs/ledger-live-common/src/families/cosmos/crypto/crypto.ts

```diff
@@ -0,0 +1,18 @@
+import cosmosBase from "./cosmosBase";
+import Cosmos from "./Cosmos";
+import Juno from "./Juno";
+import Osmosis from "./Osmosis";
+
+export default function cryptoFactory(currencyId: string): cosmosBase {
+  let res: cosmosBase;
+  if (currencyId === "osmosis" || currencyId === "osmo") {
+    res = new Osmosis();
```

Comment:
I'm more in favor of early returns, no need to create a variable here

```suggestion
    return new Osmosis();
```

Reply:
fixed